Livecheckables/john-jumbo.rb

```diff
@@ -1,4 +1,4 @@
 class JohnJumbo
-  livecheck :url   => "http://www.openwall.com/john/",
-            :regex => %r{href="j/john-([a-z0-9\.\-]+)\.t}
+  livecheck :url   => "https://github.com/magnumripper/JohnTheRipper.git",
+            :regex => /^v?(\d+(?:\.\d+)+)-Jumbo-\d$/
```

Comment:
```suggestion
            :regex => /^v?(\d+(?:\.\d+)+)-jumbo-\d$/i
```

The releases here vary between using lowercase `jumbo` and title case `Jumbo`. We want to match `jumbo` without caring about the case and we can do this by adding `i` to the end of the regex, to make it case-insensitive.

With the suggested change above, livecheck matches all the jumbo versions.

Reply:
that works for me.